src/MovieDisplay/MovieDisplay.js

```diff
@@ -0,0 +1,24 @@
+import './MovieDisplay.css';
+import MovieDetail from '../MovieDetail/MovieDetail';
+
+
+const MovieDisplay = (props) => {
+
+  return (
+    <section className='MovieDisplay' >
+      <img src={props.selectedMovie.backdrop_path} alt={props.selectedMovie.title}></img>
+      <div 
```

Comment:
We can change this to an image of a back arrow or something a bit bigger and more visually guiding to the user once we come back and clean up the layout/CSS.